tests/functional/odf-cli/test_mclock_iops_override.py

```
@@ -0,0 +1,269 @@
+        self.config_key, self.default_value = get_mclock_max_capacity_iops_config_key()
+        log.info(
+            "Using config key %s with default %s",
+            self.config_key,
+            self.default_value,
+        )
+
+        osd_pods = pod.get_osd_pods()
+        self.osd_ids = sorted([pod.get_osd_pod_id(p) for p in osd_pods])
+        log.info("Discovered OSD IDs: %s", self.osd_ids)
+
+        def finalizer():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `finalizer`

Add return type annotation: `None`

(ANN202)